great_expectations/core/util.py

```diff
@@ -533,8 +544,12 @@ def ensure_json_serializable(data):  # noqa: C901 - complexity 21
     if isinstance(data, RunIdentifier):
         return
 
-    if sqlalchemy is not None and isinstance(data, sqlalchemy.sql.elements.TextClause):
-        return str(data)
+    if sqlalchemy and isinstance(data, TextClause):
```

Comment:
Thinking about this further, I think we need to make sure that TextClause is imported. For example, in the case where we are running a version of sqlalchemy that doesn't contain TextClause. In that case sqlalchemy would be truthy but this would fail since TextClause doesn't exist.
```suggestion
    if TextClause and isinstance(data, TextClause):
```

Reply:
I do think this is a "cleaner" way of checking whether `TextClause`, but wonder how often we are going to run into a case where the `sqlalchemy` version won't contain `TextClause`...`https://docs.sqlalchemy.org/en/14/core/sqlelement.html#sqlalchemy.sql.expression.TextClause` doesn't seem to indicate that it is a new / recent feature, so maybe it has been around for a while